src/main/java/org/skriptlang/skript/lang/experiment/Feature.java

```diff
@@ -0,0 +1,83 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.lang.experiment;
```

Comment:
I think this would be better in a different spot as it isn't necessarily related to the Experiment implementation. What do you think about putting it at ch.njol.skript.registrations. Maybe under the name `DefaultExperiments`.

Reply:
> as it isn't necessarily related to the Experiment implementation

It literally _is_ the implementation of Experiment 🤣 🤣 🤣 
<img width="241" alt="image" src="https://github.com/SkriptLang/Skript/assets/14147477/e4a2dfb1-0bf4-46cf-a1a3-cfe8b596821c">

Reply:
Sorry I phrased that poorly. It isn't really part of the API for Experiments (it's just an implementation of it), so I do still think it would be better elsewhere (maybe somewhere I mentioned above)